<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>payment</class>
 <widget class="QDialog" name="payment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#payment{
background-color: #2a6496;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">#frame{
background-color: #fff;
border: none;
border-radius: 5px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Payment</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>First Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="payfirstnameLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Last Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="paylastnameLinedEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string> Parent Phone</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="payparentNumLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="paysearch_btn">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>219</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="styleSheet">
         <string notr="true">#frame_3{
background-color: #fff;
border: none;
border-radius: 5px;
}


QFrame #tpaidpushButton {
background-color: #f0ad4e;

}

QFrame #tunpaidpushButton {
background-color: #5cb85c;
}

QFrame #totalStudent_btn_3 {
background-color: #5cb85c;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="5" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="tpaidpushButton">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Total paid</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="totalPaidLabel">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="unpaidpushButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Unpaid</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="paidpushButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Paid</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Date:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="paiddateEdit">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Sitka Text</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Filter</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="tunpaidpushButton">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Total unpaid</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="totalunPaidLabel">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">#frame_2{
background-color: #fff;
border: none;
border-radius: 5px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="paymentTableWidget">
        <column>
         <property name="text">
          <string>First Name</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Last Name</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Gender</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Student Phone</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Birth Date</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Parent Phone</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Birth Date</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Address</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Option</string>
         </property>
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
